packages/@aws-cdk/aws-appmesh/test/test.virtual-node.ts

```diff
@@ -98,6 +94,170 @@ export = {
           }),
         );
 
+        test.done();
+      },
+    },
+    'when a listener is added with timeout': {
```

Comment:
There should be an empty line between tests on the same level:

```suggestion
    },

    'when a listener is added with timeout': {
```

Same comment for all new tests added.